recipe/meta.yaml

```diff
@@ -47,51 +46,57 @@ requirements:
     - zlib
   run:
     - python
-
-{% set tests_to_skip = "_not_a_real_test" %}
-# compile smoke test failing to link python3x.lib
-{% set tests_to_skip = tests_to_skip + " or test_embeddable" %}                     # [win]
-# the following needs a viewer which we don't have
-{% set tests_to_skip = tests_to_skip + " or (test_imageshow and test_show)" %}      # [linux]
-# this test is a known failure with libjpeg_turbo, but only marked as such for 2.0 upstream, not 2.x, see
-# https://github.com/python-pillow/Pillow/blob/9.3.0/Tests/test_file_libtiff.py#L885-L891
-{% set tests_to_skip = tests_to_skip + " or test_strip_ycbcr_jpeg_2x2_sampling" %}  # [not osx]
-# skip test that requires an image incorrectly flagged by AV
-{% set tests_to_skip = tests_to_skip + " or (test_tiff_crashes and crash-81154a65438ba5aaeca73fd502fa4850fbde60f8.tif)" %}
-# skip test that requires a missing mimage
-{% set tests_to_skip = tests_to_skip + " or test_tiff_mmap" %}
-# skip tests that run into troubles on win+pypy
-{% if win and python_impl == "pypy" %}
-    {% set tests_to_skip = tests_to_skip + " or test_grabclipboard" %}
-    {% set tests_to_skip = tests_to_skip + " or test_main" %}
-{% endif %}
-# #165: hard fail with no feedback on windows
-{% if win %}
-    {% set tests_to_skip = tests_to_skip + " or test_save_zero" %}
-{% endif %}
-
 test:
   imports:
     - PIL
     - PIL.Image
     - PIL.ImageCms
-  source_files:
-    # from upstream repo
-    - Tests/
-  files:
-    # from recipe; some images cannot be in upstream repo, see Tests/README
-    - Tests/images
-  requires:
-    - pytest
-    - pytest-timeout
-    # test dependency, see Tests/test_image_access.py
-    - setuptools
-    - {{ compiler('c') }}
-    # for run_test.py
-    - fsspec
-    - requests
-  commands:
-    - pytest -vv --tb=long --color=yes -k "not ({{ tests_to_skip }})" --durations=50
+
+outputs:
+  - name: pillow
```

Comment:
let's turn this into a proper output; let's move all deps, build scripts, tests etc. here, and also rename the `package:`.